src/applications/travel-pay/components/Alert.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default function Alert() {
```

Comment:
Why do we need this component for the scaffold?

Reply:
my thought was if we needed custom messaging to abstract it into our own `Alert`, but this is premature. I'll scrap it for now